pkg/operator/configobservation/configobservercontroller/observe_config_controller.go

```diff
@@ -34,11 +34,13 @@ func NewConfigObserver(
 			configobservation.Listers{
 				ImageConfigLister: configInformers.Config().V1().Images().Lister(),
 				BuildConfigLister: configInformers.Config().V1().Builds().Lister(),
+				ProxyLister:       configInformers.Config().V1().Proxies().Lister(),
```

Comment:
shouldn't we install event handlers to notice changes and update?

Reply:
you are absolutely right @sttts 

though this particular piece is a leftover from the confusion over when to use observe_config_controller.go vs. operator.go and its calls to the sync_*.go file

so I think we need
a) cleanup and full revert of the changes here in observe_config_controller.go
b) and install the event handlers as you say around here:  https://github.com/openshift/cluster-openshift-controller-manager-operator/blob/9c985c9efac3916ecaacfcecbce5c317fa8d3cfb/pkg/operator/operator.go#L52-L72

thanks ... getting started